src/app/shared/components/smart-search/smart-search.component.ts

```diff
@@ -23,9 +27,11 @@ export class SmartSearchComponent implements OnInit, AfterViewInit {
     };
 
     public filteredOptions: Observable<any[]>;
+    private TOASTR_HEADER = 'Create New Role';
 
     constructor(private route: Router,
-                private iamService: IamService) {
+        private iamService: IamService,
+        private toastr: SwitchboardToastrService) {
```

Comment:
I think informing user by toastr isn't a good idea. This adds a notification to the top right corner. I think there should go important information.